comm/src/main/scala/coop/rchain/comm/transport/StreamHandler.scala

```diff
@@ -87,15 +87,18 @@ object StreamHandler {
       }({
         // failed while collecting stream
         case (stmd, Right(Left(_))) =>
-          gracefullyClose[Task](stmd.fos).as(()) >>
+          Log[Task].warn("Failed collecting stream.") >>
+            gracefullyClose[Task](stmd.fos).as(()) >>
             stmd.path.deleteSingleFile[Task]
-        // should not happend (errors handled witin bracket) but covered for safety
+        // should not happen (errors handled within bracket) but covered for safety
         case (stmd, Left(_)) =>
-          gracefullyClose[Task](stmd.fos).as(()) >>
+          Log[Task].error("Unexpected scenario. Please pay attention.") >>
```

Comment:
This doesn't inform you in which way you can proceed. Can you add something more specific?

Reply:
https://github.com/rchain/rchain/pull/2902/commits/b94764d552d88a30e0bea77be8c951688d9024ca